core/src/main/java/cz/o2/proxima/repository/ConfigRepository.java

```diff
@@ -82,8 +82,6 @@
   private static final String TYPE = "type";
   private static final String FILTER = "filter";
 
-  private static ConfigRepository LOCAL_REPO_INSTANCE = null;
```

Comment:
This cannot be removed, because it guards that there will always be only single instance of repository in each JVM.

Reply:
```suggestion
  private static ConfigRepository LOCAL_REPO_INSTANCE = null;
```